produkt-bundles/4445-10060-Sorglos-online-TSE.md

```diff
@@ -1,18 +1,20 @@
-# fiskaltrust Sorglos Cloud TSE (in Entwicklung)
+# fiskaltrust.Sorglos mit Cloud TSE (in Entwicklung)
 
 **Produktnummer:** 4445-10060
 
+**Hinweis:** Das Sorglos mit Cloud TSE Produkt-Bundle wird angeboten, sobald eine zertifizierte Cloud-TSE am deutschen Markt verfügbar ist.
```

Comment:
"Das Sorglos mit Cloud TSE Produkt-Bundle kann leider erst angeboten werden, wenn eine zertifizierte Cloud-TSE am deutschen Markt verfügbar ist. Dies wird hoffentlich vor dem 1.10.2020 sein. Details siehe: https://docs.fiskaltrust.cloud/doc/faq/qna/market-de.html#is-there-any-certified-online-tse-available-for-germany"